ChangelogGenerator/ChangelogGenerator/ChangelogGenerator.cs

```diff
@@ -164,31 +178,77 @@ private async Task<string> GetReleaseId()
+            builder.AppendLine("| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |");
             builder.AppendLine();
-            builder.AppendLine("[Issues List]"
-                + "("
-                + "https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release="
-                + releaseId
-                + ")");
+            builder.AppendLine("<sup>1</sup> Installed with Visual Studio <VSYear> with.NET Core workload");
             builder.AppendLine();
+            builder.AppendLine(string.Format("## Summary: What's New in {0}", Options.Release));
+            builder.AppendLine();
+            OutputSection(labelSet, builder, IssueType.Feature, includeHeader:false);
+            builder.AppendLine("### Issues fixed in this release");
+            builder.AppendLine();
+            OutputSection(labelSet, builder, IssueType.DCR);
+            OutputSection(labelSet, builder, IssueType.Bug);
+            
             foreach (var key in labelSet.Keys)
             {
-                var issueTypeString = key.ToString();
-                builder.AppendLine("**" + issueTypeString + ":**");
-                builder.AppendLine();
-                foreach (var issue in labelSet[key])
+                if (key != IssueType.Feature && key != IssueType.DCR && key != IssueType.Bug)
                 {
-                    builder.AppendLine("* " + issue.Title + " - " + "[#" + issue.Number + "](" + issue.HtmlUrl + ")");
-                    builder.AppendLine();
+                    // these sections shouldn't exist. tweak the issues in github until these issues move to Feature, Bug, DCR, or go away.
+                    OutputSection(labelSet, builder, key, problem:true);
                 }
             }
 
+            builder.AppendLine("**[List of all issues fixed in this release - " + Options.Release + "]"
+            + "("
+            + "https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release="
+            + releaseId
+            + ")**");
+
             return builder.ToString();
+        }
+
+        private static void OutputSection(
+            Dictionary<IssueType, List<Issue>> labelSet,
+            StringBuilder builder,
+            IssueType key,
+            bool includeHeader = true,
+            bool problem = false)
+        {
+            
+            List<Issue> issues = null;
+            bool hasIssues = labelSet.TryGetValue(key, out issues);
 
+            if (hasIssues)
+            {
+                if (includeHeader)
+                {
+                    var issueTypeString = key.ToString();
+                    builder.AppendLine(string.Format("**{0}s:**", issueTypeString));
```

Comment:
I have noticed one problem due to appending `s` to the `IssueType`. Issues with no `Type` label are categorized under `**Nones:**` category when I generated 5.5 release notes from this branch. Can we skip adding `s` for `IssueType:None`? we can fix it manually for now.